cmake/modules/SearchRootCoreDeps.cmake

```diff
@@ -14,77 +14,100 @@ if(pyroot_experimental)
 endif()
 
 if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.14)
-  # - Check if the PYTHON_EXECUTABLE deprecated variable was passed by the
-  # user; if so, check weather it points to Python 2 or 3 and set the
-  # appropriate Python{X}_EXECUTABLE variable
-  # - Look for Python3 and set the deprecated variables to the ones set
-  # automatically by find_package(Python3 ...)
-  # - Look for Python2 and set the deprecated variables to the ones set
-  # automatically by find_package(Python2 ...) ONLY IF PYTHON3 WASN'T FOUND
-  if(PYTHON_EXECUTABLE)
-    execute_process(COMMAND ${PYTHON_EXECUTABLE} -c "import sys;print(sys.version_info[0])"
+
+  if(PYTHON_EXECUTABLE OR Python_EXECUTABLE)
```

Comment:
Why do we need `Python_EXECUTABLE` here now?

Reply:
Because Python_EXECUTABLE is the "modern" version of PYTHON_EXECUTABLE (which is deprecated)

Reply:
https://cmake.org/cmake/help/git-stage/module/FindPython.html

Reply:
(so annoying, CMake guys looks like were multiplying all possible naming cases :D )